ingestion/functions/parsing/brazil_paraiba/paraiba.py

```diff
@@ -0,0 +1,193 @@
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+if ('lambda' not in sys.argv[0]):
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+import parsing_lib
+
+_DATE_DEATH_INDEX = 6
+_GENDER_INDEX = 3
+_AGE_INDEX = 5
+_METHOD_CONFIRMATION_INDEX = 0
+_DATE_SYMPTOMS_INDEX = 2
+_MUNICIPALITY_INDEX = 1
+_PREEXISTING_CONDITIONS_INDEX = 4
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in YYYY-mm-dd format, but the date filtering API
+    expects mm/dd/YYYYZ format.
+    """
+    date = datetime.strptime(raw_date, "%Y-%m-%d")
+    return date.strftime("%m/%d/%YZ")
+
+def convert_gender(raw_gender: str):
+    if raw_gender == "Masculino":
+        return "Male"
+    elif raw_gender == "Feminino":
+        return "Female"
+    return None
+
+def convert_age(age: str):
+    if age.isdecimal():
+        return {
+            "start": float(age),
+            "end": float(age)
+        }
+    else:
+        only_age = float("".join([i for i in age if not i.isalpha()]))
+        if "dia" in age:
+            return {
+                "start": round(only_age/365.25,3), #Average number of days a year
+                "end": round(only_age/365.25,3)
+            }
+        elif "m" in age:
+            #There is one case which is reported as 1 m; I am assuming here that this also means 'mes' (month)
+            return {
+                "start": round(only_age/12,3),
+                "end": round(only_age/12,3)
+            }
+        elif "ano" in age:
+            return {
+                "start": only_age,
+                "end": only_age
+            }
+
+def convert_confirmation_method(raw_test: str):
+    if "Clínico" or "Clinico" in raw_test: #written both ways in dataset
+        return "Clinical diagnosis"
+    elif "C. Epid" in raw_test:
+        return "Other"
+    elif "Laboratorial" in raw_test:
+        return "Other"
+    elif "swab" in raw_test.lower():
+        #swabs are reported both as Swab and SWAB
+        return "PCR test"
+    elif "teste" in raw_test.lower():
+        #this is reported as both Teste Rapido and teste rapido
+        return "Serological test"  
+    else:
+        print(f'unknown confirmation method: {raw_test}')
+        return "Unknown"
+
+def convert_preexisting_conditions(raw_commorbidities: str):
+    preexistingConditions = {}
+    if raw_commorbidities != "Sem comorbidades" or "Outros" or "Tabagismo" or "Imunossupressão":
+    	#How do we want to deal with 'Others' - do we want to have hasPreexistingConditions = True
+    	#and not populate the values?
+        preexistingConditions["hasPreexistingConditions"] = True
+        commorbidities = {
+                        "Diabetes Mellitus": "diabetes mellitus", 
```

Comment:
I recommend to run autopep8 on your code (it's a simple plugin in vscode if you use it) to format the code nicely.